code/modules/cm_marines/smartgun_mount.dm

```diff
@@ -44,6 +44,8 @@
 	icon_state = "M56D_gun_e"
 	var/rounds = 0 // How many rounds are in the weapon. This is useful if we break down our guns.
 	var/has_mount = FALSE // Indicates whether the M56D will come with its folding mount already attached
+	var/defense_check_range = 2
+	var/can_be_near_defense = FALSE
```

Comment:
This variable seems pointless if all you're going to do is check `if(!can_be_near_defense)` and it never changes may as well not have it and remove those checks also.